ui/admin/app/components/form/credential-store/static/index.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { options } from 'api/models/credential-store';
+
+export default class FormStaticCredentialStoreIndexComponent extends Component {
+  // =properties
+  /**
+   * @type {object}
+   */
+  types = options;
+  icon = options[this.args.model.type].icon;
```

Comment:
I am a bit confused here. Why we are importing credential-store model for `options` (which I think is fine) but then using `this.args.model.type` to get the icon type? Isn't the same model?
I have the feeling we are getting that model twice using different patterns, I might be out of context.

Reply:
I need to pass in the model to get the icon. (this is for info field component) But based on Zed's comment below, I'll explicitly write the type to grab the icon